src/main/java/com/atlassian/bitbucket/jenkins/internal/deployments/DeploymentStepDescriptorHelper.java

```diff
@@ -0,0 +1,104 @@
+package com.atlassian.bitbucket.jenkins.internal.deployments;
+
+import com.atlassian.bitbucket.jenkins.internal.model.deployment.BitbucketDeploymentEnvironmentType;
+import com.atlassian.bitbucket.jenkins.internal.provider.JenkinsProvider;
+import com.google.inject.Singleton;
+import hudson.model.Item;
+import hudson.util.FormValidation;
+import hudson.util.ListBoxModel;
+import jenkins.model.Jenkins;
+
+import javax.annotation.CheckForNull;
+import javax.inject.Inject;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.Arrays;
+import java.util.Comparator;
+
+import static org.apache.commons.lang3.StringUtils.isBlank;
+
+@Singleton
+public class DeploymentStepDescriptorHelper {
+
+    private static final FormValidation FORM_VALIDATION_OK = FormValidation.ok();
+    private static final int ENVIRONMENT_NAME_MAX = 255;
+    private static final int ENVIRONMENT_URL_MAX = 1024;
+    private static final int ENVIRONMENT_KEY_MAX = 255;
```

Comment:
total nit, but ordering alphabetically is nice. If you make other changes then maybe update :D